README.md

```diff
@@ -2,17 +2,40 @@
 
 > In this project, I will use a relational database to create the initial data structure for a vet clinic. I will create a table to store animals'' information, insert some data into it, and query it.
 
-
 ## Getting Started
 
 This repository includes files with plain SQL that can be used to recreate a database:
 
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
-- 
+- [schema.sql](./schema.sql)  where I created all tables.
+- [data.sql](./data.sql) to populate tables with sample data.
+- [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. 
+
 <img width="705" alt="Screen Shot 2022-07-11 at 19 07 24" src="https://user-images.githubusercontent.com/91237525/178329833-b85227dc-ea82-406b-b789-6d3e913a18e5.png">
 
+<img width="647" alt="Screen Shot 2022-07-11 at 21 51 37" src="https://user-images.githubusercontent.com/91237525/178362524-dff22f66-7753-4649-b3e8-0f47f37a2fd7.png">
+<img width="627" alt="Screen Shot 2022-07-11 at 22 07 14" src="https://user-images.githubusercontent.com/91237525/178362528-ed0eaa68-6810-4fd5-8a36-418eba33ee07.png">
+<img width="619" alt="Screen Shot 2022-07-11 at 22 08 25" src="https://user-images.githubusercontent.com/91237525/178362530-f7aa7414-92b3-4d4d-ad32-24c934789e1d.png">
+<img width="622" alt="Screen Shot 2022-07-11 at 22 09 18" src="https://user-images.githubusercontent.com/91237525/178362533-2d213418-7152-4e64-bdb2-d83ada8f6bb8.png">
+<img width="868" alt="Screen Shot 2022-07-11 at 22 11 16" src="https://user-images.githubusercontent.com/91237525/178362535-c532ec22-ae12-48ed-9568-26050aab1048.png">
+<img width="705" alt="Screen Shot 2022-07-11 at 22 26 25" src="https://user-images.githubusercontent.com/91237525/178362537-e0515909-c57b-4f69-b3df-f85daaef5583.png">
+<img width="853" alt="Screen Shot 2022-07-12 at 20 26 44" src="https://user-images.githubusercontent.com/91237525/178600083-368ce268-c0dc-47d3-bace-6760becd4efd.png">
+<img width="853" alt="Screen Shot 2022-07-12 at 20 46 11" src="https://user-images.githubusercontent.com/91237525/178600084-493921f9-cc98-4683-b19a-872122246d9a.png">
+<img width="856" alt="Screen Shot 2022-07-12 at 20 53 28" src="https://user-images.githubusercontent.com/91237525/178600087-6a385d3c-f3f9-4b53-b4f6-93228d15f9c9.png">
```

Comment:
- [ ] As requested by the previous reviewer, please add these screenshots to the pull request description instead of the README file. You can edit the pull request description and add it

Reply:
Oh okay thank you so much!